packages/gatsby/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "../../tsconfig.json",
-  // This is for typegen purposes only. For now index.d.ts is manually-created, but gatsby/internal is auto-generated
-  "include": ["./src/internal.ts"]
+  // For now index.d.ts is manually-created, but gatsby/internal is auto-generated
+  "include": ["src/**/*.ts"],
+  "exclude": ["**/__tests__/**/*"]
```

Comment:
does this mean we generate bunch of `.d.ts` files now as part of build? Should we maybe have separate tsconfig file (same as it is today) for `internal.d.ts` generation and adjust `build:types` `tsc` command to use that config?

Reply:
Yeah, we'll generate `.d.ts` files for those, too. I don't see any harm in that, we do that also for other packages like `create-gatsby` or `gatsby-plugin-image`. Is there a specific reason you wouldn't want that?

Reply:
dunno - I thought there was a reason to only generate one for `internal.ts`, but if it works we can keep it as you have right now